### ISSUE TITLE
Fix Undeclared in Pharo 13

### ISSUE THREAD

**Issue by demarey** [hpi-swa/smalltalkCI#648]
Because of recent changes in Pharo 13, Smalltalk CI fails because it uses a class that has been renamed.
The fix is to add this method in `SmalltalkCIPharo13`:
```smalltalk
imageInfo
	^ String streamContents: [ :stream | StSystemReporterPresenter new reportImage: stream ]
```

I did a PR but metadata writing from Pharo is not the same as in Squeak and there was a lot of noise.
Tell me if you prefer the PR with metadata noise or if this issue is enough.

**Comment by fniephaus** [comment]
Hi @demarey, a PR would be nice. Maybe create it manually but adding a file for the method and adding it to the [corresponding properties file](https://github.com/hpi-swa/smalltalkCI/blob/master/repository/SmalltalkCI-Pharo-Core.package/SmalltalkCIPharo13.class/methodProperties.json)?

**Comment by fniephaus** [comment]
cc @theseion

**Comment by demarey** [comment]
done in #649

**Comment by theseion** [comment]
Fixed by #646
